### PR TITLE
 FEATURE: More descriptive error message on personal schedule and Button! 

### LINKED ISSUE #22: FEATURE: More descriptive error message on personal schedule
### Have you updated the title?

- [X] I have updated the issue's title.

### As a...

user

### I can...

get a better error message when adding a course with no schedule

### So that...

I can understand why it's not working

### Discussion

Currently, adding a course with no personal schedule available just returns the text "Error.". Please replace this with a more descriptive error message so that the user knows that they have to create a personal schedule first.

OPTIONAL BONUS: If this happens, give the user a way to create a new schedule without having to find the page themselves -- either link to the "Create Schedule" page or create a modal/pop-up to quickly create a schedule without leaving the page (worth more points).

### Acceptance Criteria

- [x] Trying to add a course to a schedule with no schedule created should return a more descriptive error message which describes the actual issue
- [x] [BONUS] The user can easily create a new schedule if this error is found.

### Implementation Todos

Frontend:
- [x] Replace the "Error" message with a more descriptive error message.
- [x] [BONUS] Add a link to the create schedule page OR a modal / pop up to quickly create a schedule.

Testing:
- [x] Front end tests pass and there is adequate coverage
- [x] Back end tests pass and there is adequate coverage

### PR DESCRIPTION
**Pull request by El-Beetor**
Added a more descriptive error message for when a user tries to add a course to their schedule but they 
did not include a schedule. 

I also created a button that links to the create schedule page.(for the bonus)

image

closes #22

### CODE REVIEW

**Review by tallyhawley — CHANGES_REQUESTED** [review]
Dokku link?

**Comment by El-Beetor** [comment]
Dokku Link: https://project-el-beetor.dokku-05.cs.ucsb.edu/

**Review by tallyhawley — CHANGES_REQUESTED** [review]
This error message shouldn't display on ALL errors. Only on the error of "No personal schedule". 

This is not the expected behavior:
image

I should get the existing `mutation.error.response.data?.message` which references an invalid enroll code.

**Comment by El-Beetor** [comment]
Try it again. Changed the error handling
image

File: frontend/src/main/pages/Courses/PSCourseCreatePage.js [hunk 1]
```diff
@@ -41,23 +41,52 @@ export default function CoursesCreatePage() {
     return <Navigate to="/courses/list" />;
   }
   if (mutation.isError) {
-    return (
-      <BasicLayout>
-        <div className="pt-2">
-          <h1>Create New Course</h1>
+    const errorMessage =
+      mutation.error.response.data?.message || "Unkown Error";
```
Comment (tallyhawley) [inline frontend/src/main/pages/Courses/PSCourseCreatePage.js]: typo

**Review by tallyhawley — CHANGES_REQUESTED** [review]
Looks good other than the typo, just fix that and you'll be good to merge :)

**Review by tallyhawley — APPROVED** [review]
Looks good to me

**Review by brvola — APPROVED** [review]
LGTM!

**Pull request merged**